src/utils/component-versioning.js

```diff
@@ -18,24 +18,34 @@ const extractDependencies = componentClass => {
 const oldTag = (tagName, closing = '') => `<${closing}${tagName}`;
 
 const newTag = (tagName, aVersion, closing) =>
-  oldTag(versionedTag(tagName, aVersion), closing);
+  oldTag(`${versionedTag(tagName, aVersion)} `, closing);
 
 // Example: someStrings = ['<div><axa-dropdown .items="','" </axa-dropdown></div>']
 //          aTagname = 'axa-dropdown', aVersion = '7.0.2'
 //
 // Rewritten as           ['<div><axa-dropdown-7-0-2 .items="','" </axa-dropdown-7-0-2></div>'].
 //
-// Note: this uses the split-join technique to perform global string substitution
-// without needing the special-character escaping necessary
-// a reg-exp-based alternative (the latter is marginally faster, but our strings here are short)
-const rewrite = (someStrings, aTagName, aVersion) =>
-  someStrings.map(string =>
+// The first character after each tag name (e.g. after "<axa-button-link") can be either a line
+// break or a whitespace. We also need to consider that charater, because "<axa-button" is part
+// of "<axa-button-link" (which was a prod issue) and generated a version like this: "<axa-button-6.0.5-link"
+// Reason: It was not made sure that the tagname is "finished" before, just if it is contained.
+//
+// Note: This uses the split-join technique to perform global string substitution without needing the
+// special-character escaping necessary a reg-exp-based alternative (the latter is marginally faster,
+// but our strings here are short)
+const rewriteTagsWithVersion = (someStrings, aTagName, aVersion) => {
+  const tagNameWithAddedWhitespace = `${aTagName} `;
+  const versionWithAddedWhitespace = `${aVersion} `;
+  return someStrings.map(string =>
     string
-      .split(oldTag(aTagName))
-      .join(newTag(aTagName, aVersion))
-      .split(oldTag(aTagName, '/'))
-      .join(newTag(aTagName, aVersion, '/'))
+      .split('\n')
+      .join(' ')
```

Comment:
Maybe change that to regexp like we talked about. That PR should be reviewed by @markus-walther